cypress/local-only/tests/integration/fileUpload/upload-court-issued-pdf-validation.cy.ts

```diff
@@ -0,0 +1,98 @@
+import { goToCase } from '../../../../helpers/caseDetail/go-to-case';
+
+import { MAX_FILE_SIZE_MB } from '../../../../../shared/src/business/entities/EntityConstants';
+import { attachFile } from '../../../../helpers/file/upload-file';
+import { loginAsDocketClerk1 } from '../../../../helpers/authentication/login-as-helpers';
+
+describe('upload court issued document validations', () => {
+  const docketNumber = '102-67'; // Any existing docket number works
+
+  beforeEach(() => {
+    loginAsDocketClerk1();
+    goToCase(docketNumber);
+    cy.get('[data-testid="case-detail-menu-button"]').click();
+    cy.get('[data-testid="menu-button-upload-pdf"]').click();
+  });
+
+  it('should display error modal when a non-pdf file is selected and see no file selected after closing modal', () => {
+    cy.intercept('POST', '/logError').as('logErrorRequest');
+
+    attachFile({
+      filePath: '../../helpers/file/non-pdf.txt',
+      selector: '[data-testid="primary-document-file"]',
+    });
+    cy.wait('@logErrorRequest');
+
+    cy.get('@logErrorRequest.all').should('have.length', 1);
+    cy.get('[data-testid="file-upload-error-modal"]').contains(
+      'The file is not a PDF. Select a PDF file or resave the file as a PDF.',
+    );
+    cy.get('[data-testid="modal-button-confirm"').click();
+    cy.get('[data-testid^="upload-file-success"]').should('not.exist');
+  });
+
+  it('should display error modal when a non-supported pdf format is selected', () => {
+    cy.intercept('POST', '/logError').as('logErrorRequest');
+
+    attachFile({
+      filePath: '../../helpers/file/corrupt-pdf.pdf',
+      selector: '[data-testid="primary-document-file"]',
+    });
+    cy.wait('@logErrorRequest');
+
+    cy.get('@logErrorRequest.all').should('have.length', 1);
+    cy.get('[data-testid="file-upload-error-modal"]').contains(
+      'The file is corrupted or in an unsupported PDF format. Ensure that the file is not corrupted and/or is in a supported PDF format and try again.',
+    );
+  });
+
+  it('should display error modal when a password-protected pdf is selected', () => {
+    cy.intercept('POST', '/logError').as('logErrorRequest');
+
+    attachFile({
+      filePath: '../../helpers/file/password-protected-pdf.pdf',
+      selector: '[data-testid="primary-document-file"]',
+    });
+    cy.wait('@logErrorRequest');
+
+    cy.get('@logErrorRequest.all').should('have.length', 1);
+    cy.get('[data-testid="file-upload-error-modal"]').contains(
+      'The file is encrypted or password protected. Remove encryption or password protection and try again.',
+    );
+  });
+
+  it('should display error modal when a file larger than the limit is selected', () => {
+    cy.intercept('POST', '/logError').as('logErrorRequest');
+    const largeFile = new Blob(
+      [new ArrayBuffer((MAX_FILE_SIZE_MB + 1) * 1024 * 1024)],
+      {
+        type: 'application/pdf',
+      },
+    );
```

Comment:
I definitely appreciate that we're not adding a 251MB file into the repo for this!